test/show.jl

```diff
@@ -278,20 +278,19 @@ end
 let oldout = STDOUT, olderr = STDERR
     local rdout, wrout, rderr, wrerr, out, err, rd, wr
     try
-        rd, wr = redirect_stdout()
-        @test dump(STDERR) == nothing
```

Comment:
it wasn't checking it's output so it's just artificially inflating the coverage. I added more than enough tests below of the same to make up for it.